astropy/modeling/fitting.py

```diff
@@ -1120,6 +1120,10 @@ class _NonLinearLSQFitter(Fitter):
     calc_uncertainties : bool
         If the covariance matrix should be computed and set in the fit_info.
         Default: False
+    check_non_finite : bool
```

Comment:
I think this line should be moved to after `use_min_max_bounds` to be consistent with the constructor signature?

Also do you plan to add change log? Thanks!